### ISSUE TITLE
CarPlay route lines disappear

### ISSUE THREAD

**Issue by Raphty** [osmandapp/OsmAnd-iOS#2007]
During navigation the line disappears periodically. Sometimes for longer segments and sometimes only short ones, but this is making it hard to follow.

**Comment by vshcherb** [comment]
The line disappears behind my location or in front?

**Comment by Raphty** [comment]
it disappears in front! 
if it would be behind , i would not care.

**Comment by Raphty** [comment]
I switched to the TestFlight version, and haven't seen this issue again. 

I think it can be considered solved.

**Comment by Zirochkabila** [comment]
OsmAnd version 3.0 - The case was not reproduced.
The route line is present on the map during the entire navigation.
